router/rootfs/etc/confd/templates/backend.conf

```diff
@@ -0,0 +1,3 @@
+proxy_ssl_verify        off;
```

Comment:
I could probably just get rid of `backend.conf` completely and move `proxy_ssl_verify        off;` into `nginx.conf`. Should I bother?